models/record.go

```diff
@@ -163,40 +188,65 @@ func (rc *RecordConfig) UnmarshalJSON(b []byte) error {
 		Original  interface{}       `json:"-"` // Store pointer to provider-specific record object. Used in diffing.
 
 		MxPreference     uint16            `json:"mxpreference,omitempty"`
+
 		SrvPriority      uint16            `json:"srvpriority,omitempty"`
 		SrvWeight        uint16            `json:"srvweight,omitempty"`
 		SrvPort          uint16            `json:"srvport,omitempty"`
+
 		CaaTag           string            `json:"caatag,omitempty"`
 		CaaFlag          uint8             `json:"caaflag,omitempty"`
+
 		DsKeyTag         uint16            `json:"dskeytag,omitempty"`
 		DsAlgorithm      uint8             `json:"dsalgorithm,omitempty"`
 		DsDigestType     uint8             `json:"dsdigesttype,omitempty"`
 		DsDigest         string            `json:"dsdigest,omitempty"`
+
 		LocVersion       uint8             `json:"locversion,omitempty"`
 		LocSize          uint8             `json:"locsize,omitempty"`
 		LocHorizPre      uint8             `json:"lochorizpre,omitempty"`
 		LocVertPre       uint8             `json:"locvertpre,omitempty"`
 		LocLatitude      int               `json:"loclatitude,omitempty"`
 		LocLongitude     int               `json:"loclongitude,omitempty"`
 		LocAltitude      uint32            `json:"localtitude,omitempty"`
+
+		LocLatDegrees    uint8             `json:"loclatdegrees,omitempty"`
+		LocLatMinutes    uint8             `json:"lotlatminutes,omitempty"`
+		LocLatSeconds    float32           `json:"loclatseconds,omitempty"`
+		LocLatDirection  string            `json:"loclatdirection,omitempty"`
+		LocLongDegrees   uint8             `json:"loclongdegrees,omitempty"`
+		LocLongMinutes   uint8             `json:"lotlongminutes,omitempty"`
+		LocLongSeconds   float32           `json:"loclongseconds,omitempty"`
```

Comment:
I couldn't see the LOC records at cloud flare link. So we assume only seconds will take decimal fractions ie floats. Degrees and minutes are whole integers.